tcms/telemetry/static/telemetry/js/testing/status-matrix.js

```diff
@@ -0,0 +1,117 @@
+$(document).ready(() => {
+
+    loadInitialProduct();
+
+    document.getElementById('id_product').onchange = () => {
+        update_version_select_from_product();
+        update_build_select_from_product();
+        updateTestPlanSelectFromProduct();
+
+        redrawTable();
+    };
+
+    document.getElementById('id_version').onchange = redrawTable;
```

Comment:
- select product + version
- then back to \-\-\-
- results table is not redrawn